Testing/Unit/Python/sitkExternalViewerTest.py

```diff
@@ -70,7 +70,8 @@ def test_image_viewer(self):
 
         try:
             viewer = sitk.ImageViewer()
-            viewer.SetTitle("Basic Image Viewer Test")
+            # Some funky characters in the title to test the temp file name filter
+            viewer.SetTitle("Basic Image Viewer Test/:\'")
```

Comment:
Is there a GetTitle method that could be used to verify the operation completed as expected? Perhaps testing a couple cases?

Reply:
It doesn't filter the title string.  It only filters the temporary file name string, which we don't provide access to.